src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3408,7 +3408,7 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
                     // Check for the nested modules with "."
                     // Example: from x.y import z
                     if (mod_sym.find(".") != std::string::npos) {
-                        mod_sym.replace(mod_sym.find("."), 1, "/");
+                        mod_sym = replace(mod_sym, "[.]", "/");
```

Comment:
Seems like this was hard coded before.

Reply:
Previously, I thought `replace()` would replace every occrurances in the string. Now, I realized that it replaces only the 1st occurance or simple the index of the `fine()`.